spec/unit/installer/analyzer_spec.rb

```diff
@@ -672,84 +672,22 @@ module Pod
 
       #-------------------------------------------------------------------------#
 
-      it 'handles test only pod targets' do
```

Comment:
tests no longer applicable as `pod_target_test_only?` was removed as a method of the `analyzer.rb`